include/RAJA/util/basic_mempool.hpp

```diff
@@ -423,7 +423,7 @@ class MemPool {
 private:
   using arena_container_type = std::list<detail::MemoryArena>;
 
-#if defined(RAJA_ENABLE_OPENMP) && defined(_OPENMP)
```

Comment:
We might want to add a check to `config.hpp` that checks if `_OPENMP` is defined when `RAJA_ENABLE_OPENMP` is, and error if not. This would stop people building RAJA with OpenMP, but then compiling code without the correct flags to enable it.

Reply:
We might even want to take that out of the hands of cmake and make RAJA_ENABLE_OPENMP an alias for _OPENMP.  We would have to get rid of the last of the compiled code that relies on OpenMP, but if we could it would be one less potential inconsistency.

Reply:
@trws I like the alias idea. 👍